docs/source/multi_gpu.rst

```diff
@@ -593,7 +597,7 @@ Below are the possible configurations we support.
 
 Implement Your Own Distributed (DDP) training
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-If you need your own way to init PyTorch DDP you can override :meth:`pytorch_lightning.plugins.ddp_plugin.DDPPlugin.init_ddp_connection`.
+If you need your own way to init PyTorch DDP you can override: :meth:`pytorch_lightning.plugins.ddp_plugin.DDPPlugin.init_ddp_connection`.
```

Comment:
```suggestion
If you need your own way to init PyTorch DDP you can override :meth:`~pytorch_lightning.plugins.ddp_plugin.DDPPlugin.init_ddp_connection`.
```
it's part of the sentence, so the colon should not be necessary? But I suggest to shorten the name with the tilde.